.github/workflows/release.yaml

```diff
@@ -23,6 +17,39 @@ jobs:
         with:
           go-version: "1.17.x"
 
+      - name: Build tagged release, elf, debug
+        id: build-debug-elf
+        uses: oasisprotocol/oasis-sdk/.github/actions/hash-rust@main
+        with:
+          image: oasisprotocol/runtime-builder:main
+          pkg-dirs: runtime
+          binaries: sapphire-paratime
+          clean: no
+          setup: |
+            export OASIS_UNSAFE_SKIP_AVR_VERIFY=1 OASIS_UNSAFE_SKIP_KM_POLICY=1 OASIS_UNSAFE_ALLOW_DEBUG_ENCLAVES=1
+
+      - name: Build tagged release, sgxs, debug
```

Comment:
Any reason for also including a debug SGXS build? I assume nobody will sign this? You can skip the SGXS variant if this is only used for local testing (just omit the `--sgx-executable` when building an ORC.

Reply:
it'd be cool to someday have a key in CI that auto-signs this debug build in a way that's insecure but convenient. Then it'd be possible to test in real SGX

Reply:
You don't need to sign debug builds, Oasis Core will do it for you using a known (and blacklisted in production) signer if configured in unsafe mode. If you think debug SGX builds are also useful, use it as-is.

Reply:
Oh, also if you want debug SGX builds, the other problem is that the `consensus_trust_root` binds the runtime to the given consensus layer deployment so this will not work in a local environment.